### PR TITLE
[22.03] passh: an sshpass alternative

### PR DESCRIPTION
**Pull request by clarkwang**
Sshpass does not work well in a few scenarios. Passh is an sshpass
alternative which tries to fix the issues.

Signed-off-by: Clark Wang <dearvoid@gmail.com>

Maintainer: Clark Wang <dearvoid@gmail.com>
Compile tested: 21.02.3-ramips-mt7621
Run tested: 21.02.3-ramips-mt7621 (Netgear R6220)

Description:

### CODE REVIEW

File: utils/passh/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+PKG_NAME    := passh
+PKG_VERSION := 1.0.1
+PKG_RELEASE := $(AUTORELEASE)
+
+PKG_SOURCE     := $(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL := https://codeload.github.com/clarkwang/passh/tar.gz/v$(PKG_VERSION)?
+PKG_HASH       := f6efc7127515b3716108fa28439caca561d16923404bb1a8f734b41340b3f14e
+
+PKG_MAINTAINER    := Clark Wang <dearvoid@gmail.com>
+PKG_LICENSE       := GPL-3.0
```
Comment (BKPepe) [inline utils/passh/Makefile:13]: This is old SPDX License Identifier.
Please see: https://spdx.org/licenses/

**Review by BKPepe — CHANGES_REQUESTED** [review]
Same as https://github.com/openwrt/packages/pull/18332#pullrequestreview-950258582

**Comment by clarkwang** [comment]
Hi BKPepe,

I've updated the license and added a test.sh.

**Comment by BKPepe** [comment]
Thanks! Please split the changes to the separate commit and send it to the master first

**Comment by clarkwang** [comment]
Has committed to master first and cherry-picked to 22.03 branch.

**Pull request merged**